src/main/java/com/genfood/foodgenback/endpoint/rest/model/SignUpRequest.java

```diff
@@ -0,0 +1,14 @@
+package com.genfood.foodgenback.endpoint.rest.model;
+
+import java.io.Serializable;
+import lombok.Builder;
+import lombok.Data;
+
+@Builder
+@Data
+public class SignUpRequest implements Serializable {
```

Comment:
Update this class to look like the specification.

![image](https://github.com/HEI-CleanCodeProject/FoodGen-Back/assets/98638893/831f737f-a2b8-4f56-93c2-793c371ece0f)